### PR TITLE
ISSUE-2472: namespace -> cluster, tenant -> tenant_id

### LINKED ISSUE #2742: Consider renaming the namespace to cluster_name
**Summary**
Consider renaming the `namespace` to `cluster_name`(contains the config item name, variables name, struct name in code)

### LINKED ISSUE #2746: Consider renaming the tenant to tenant_id
**Summary**
https://github.com/datafuselabs/databend/issues/2742#issuecomment-966082920
Consider renaming the `tenant` to `tenant_id`(contains the config item name, variables name, struct name in code)

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

This PR mainly change the config/query field:
1. namespace -> cluster_id
2. tenant -> tenant_id

## Changelog

- Improvement

## Related Issues

Fixes #2742 
Fixes #2746 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by BohuTANG** [comment]
@ZhiHanZ some docker config changed, please take a review

**Review by ZhiHanZ — APPROVED** [review]

**Comment by ZhiHanZ** [comment]
LGTM, in alpha stage, we do not have to setup version backward compatibility

File: cli/src/cmds/clusters/add.rs [hunk 1]
```diff
@@ -160,19 +160,19 @@ impl Command for AddCommand {
                     .default_value("2"),
             )
             .arg(
-                Arg::new("query_namespace")
-                    .long("query-namespace")
-                    .env(databend_query::configs::config_query::QUERY_NAMESPACE)
+                Arg::new("query_cluster_id")
+                    .long("query-cluster-id")
```
Comment (flaneur2020) [inline cli/src/cmds/clusters/add.rs:164]: how to get the numeric cluster id value on startup the query instance?  q.q

in etcd there's a `--initial-cluster-token` argument, which gives the cluster a human readable name
https://etcd.io/docs/v3.2/op-guide/configuration/#--initial-cluster-token

a human readable name is also valuable on setting the monitoring metrics, we can get some alerts about `cluster biz-etl01 loads too high` by the extracting the `cluster` tag inside the metric

**Review by zhang2014 — APPROVED** [review]

File: cli/src/cmds/clusters/add.rs (hunk 1 above)
Comment (BohuTANG) [reply cli/src/cmds/clusters/add.rs:164]: cluster-id is a string(uuid) style not numberic, but it can be set as "12345"

Comment (flaneur2020) [reply cli/src/cmds/clusters/add.rs:164]:
```
                    .long("query-cluster-id")
```
oh sorry i missunderstood the `.long("")` method means the longer args name, not means `long` as args type, ignore me >.<

**Pull request merged**